File: backend/hot-place-service/src/main/java/com/hotspot/PopplarApplication.java
package com.hotspot;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;


@SpringBootApplication
@EnableScheduling
public class PopplarApplication {

	public static void main(String[] args) {
		SpringApplication.run(PopplarApplication.class, args);
	}

}

File: backend/member-service/src/main/kotlin/com/hotspot/PopplarApplication.kt
package com.hotspot

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@SpringBootApplication
@EnableJpaAuditing
class PopplarApplication

fun main(args: Array<String>) {
	runApplication<PopplarApplication>(*args)
}
